test/e2e/flowexporter_test.go

```diff
@@ -238,5 +252,112 @@ func checkRecordsWithPodIPs(t *testing.T, data *TestData, podAIP string, podBIP
 	assert.Equal(t, templateRecords, clusterInfo.numNodes, "Each agent should send out template record")
 	// Single iperf resulting in two connections with separate ports. Suspecting second flow to be control flow to exchange
 	// stats info. As 5s is export interval and iperf traffic runs for 10s, we expect 4 records.
+	assert.GreaterOrEqual(t, dataRecordsCount, 4, "Iperf flow should have expected number of flow records")
+}
+
+func checkRecordsWithServiceIP(t *testing.T, data *TestData, podAIP string, serviceIP string, isIPv6 bool) {
```

Comment:
Can this function be merged with `checkRecordsWithPodIPs` by calling the function as `checkRecordsForFlows` or `checkRecordsWithFlowFilter`? We can call the podIP args as `srcIP` and `dstIP` to be generic to fit serviceIP. Use `checkService bool` to check for relevant info. 
This will help check the bandwidth info for Pod-To-Service flows too or skip with `checkBandwidth bool`. 
What do you think?

Reply:
Makes sense to me. It will also help save some duplication of code. Thanks!